common/coat_of_arms/coat_of_arms/wc_religious_titles.txt

```diff
@@ -130,4 +130,245 @@ d_bronze_wyrmcult = {
 		color2 = rgb { 205 127 50 }
 		color3 = rgb { 205 127 50 }
 	}
-}
\ No newline at end of file
+}
+
+### Pandaren
+k_shado_pan = { # Shado-Pan Monks by Juke
+	pattern="pattern_solid.dds"
+	color1=rgb { 154 43 4 }
+	color2=yellow
+	color3=blue
+	colored_emblem={
+		color1=yellow
+		texture="ce_tamgha_buduyan.dds"
+		instance={
+			position={ 0.500000 0.130000 }
+			scale={ 0.775000 -0.220000 }
+		}
+
+	}
+
+	colored_emblem={
+		color1=yellow
+		texture="ce_tamgha_adlari.dds"
+		instance={
+			position={ 0.500000 0.800000 }
+			scale={ 0.430000 0.430000 }
+			depth=4.010000
+		}
+
+	}
+
+	colored_emblem={
+		color1=yellow
+		texture="ce_rune_orc_03.dds"
+		instance={
+			position={ 0.720000 0.305000 }
+			scale={ 0.440000 -0.440000 }
+			depth=1.010000
+			rotation=133
+		}
+
+		instance={
+			position={ 0.275000 0.305000 }
+			scale={ 0.440000 0.440000 }
+			depth=2.010000
+			rotation=45
+		}
+
+	}
+
+	colored_emblem={
+		color1=rgb { 206 142 48 }
+		color2=red
+		color3=white
+		texture="ce_rune_orc_17.dds"
+		instance={
+			position={ 0.770000 0.595000 }
+			scale={ -0.420000 0.420000 }
+			depth=5.010000
+			rotation=12
+		}
+
+		instance={
+			position={ 0.240000 0.595000 }
+			scale={ 0.420000 0.420000 }
+			depth=6.010000
+			rotation=345
+		}
+
+	}
+
+	colored_emblem={
+		color1=yellow
+		texture="ce_islamic_lis.dds"
+		instance={
+			position={ 0.500000 0.535000 }
+			scale={ 0.500000 -0.380000 }
+			depth=3.010000
+		}
+
+	}
+
+}
+k_golden_lotus = { # Golden Lotus by Juke (Bad)
+	pattern="pattern_solid.dds"
+	color1=rgb { 184 159 142 }
+	color2=white
+	color3=purple
+	colored_emblem={
+		color1=rgb { 251 168 46 }
+		texture="ce_boat.dds"
```

Comment:
do you think this could be a lotus instead?
There's a vanilla CoA for that

Reply:
Not sure how i didnt think of that haha! Thats supposed to be the Golden Pagoa I believe

Reply:
![image](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/assets/76008882/9d4b0112-6224-43cd-8eaa-eb41ce172738)

Made a better one! I think..